src/scheduler/agent/database.c

```diff
@@ -1098,27 +1147,15 @@ char* get_email_command(scheduler_t* scheduler, char* user_email)
         g_string_append_printf(client_cmd, " -S smtp-auth=plain");
```

Comment:
Seems like new `s-nail` does not like empty password. So if `SMTPAuth` is set to `N` (None), ` -S smtp-auth=none` needs to be added.